assets/js/base/components/price-slider/index.js

```diff
@@ -212,9 +213,27 @@ const PriceSlider = ( {
 			) {
 				return;
 			}
+
 			const isMin = event.target.classList.contains(
 				'wc-block-price-filter__amount--min'
 			);
+
+			// When user insert in the minimum price input a value greater than the max price,
+			// we update the value of the slider to 0 - max.
```

Comment:
Can't we just reject the min inserted and default to the previous min instead of reseting max as well? this is more a UX thing and not code related but if something changes, just reset it instead of reseting two things.

Reply:
> Can't we just reject the min inserted and default to the previous min instead of reseting max as well? this is more a UX thing and not code related but if something changes, just reset it instead of reseting two things.

My comment is probably unclear. For `max`, I mean the current max price. So what my code does it is just reset to 0 the minimum price. 

```suggestion
			// When user insert in the minimum price input a value greater than the max price,
			// we set to 0 the minimum price.
```

Reply:
Yes, this is great!

Reply:
I rephrased it again because I just re-checked the logic better.
```
// When the user inserts in the max price input a value less or equal than the current minimum price,
// we set to 0 the minimum price.
 ```